src/Symfony/Component/HttpClient/Response/AmpResponse.php

```diff
@@ -346,7 +323,7 @@ private static function followRedirects(Request $originRequest, AmpClientState $
                 $request->setBody(AmpBody::rewind($response->getRequest()->getBody()));
             }
 
-            foreach ($originRequest->getRawHeaders() as [$name, $value]) {
+            foreach ($originRequest->getHeaderPairs() as [$name, $value]) {
                 $request->setHeader($name, $value);
```

Comment:
This should likely call `addHeader` instead?

Reply:
indeed, thanks